CHANGELOG.md

```diff
@@ -1,19 +1,20 @@
 # 0.3.0 - 2018-06-07
 
-- Fix deadlock during prevote migration process
-- Keep compatible with protobuf 2.0
-- Make raft `Send`
-- Add more docs and example
-- Support configuring election timeout range
+- Now supports configuring the election timeout range (https://github.com/pingcap/raft-rs/pull/63).
+- Fixed a compiler error (https://github.com/pingcap/raft-rs/pull/64)
```

Comment:
I think both this and "Workaround a problem" are parts of "keep compatible with rust-protobuf 2.0", which seems more clear.

Reply:
:+1: